src/camvidlog2/yoloe/data.py

```
@@ -0,0 +1,38 @@
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)